### ISSUE TITLE
paper dialog does not show filters

### ISSUE THREAD

**Issue by cutout33** [Collaborne/paper-search#1]
Hi,

am trying to use the paper-search-dialog in your demo but am unable to, when I click the open button the dialog shows up but without any filters, in the console I get the following error:
`Uncaught TypeError: this.$.filtersList.fire is not a function
`
am unable to install the component throgh bower, I get:
`
bower                            retry Request to https://bower.herokuapp.com/packages/paper-search failed with ECONNRESET`

Also the component seems to only work on chrome and opera, I tried firefox, safari and edge but it does not show filters too...

**Comment by ronnyroeller** [comment]
@cutout33 Thanks for the feedback.

Which browser (and browser version) are you using when you see the `Uncaught TypeError`?

The component is correctly registered and I can also install it without any issues:

```
$ bower lookup paper-search
paper-search git://github.com/Collaborne/paper-search.git
```

This might help you further: http://stackoverflow.com/questions/21750804/bower-calls-blocked-by-corporate-proxy

**Comment by ronnyroeller** [comment]
@cutout33 Turns out that one of the recent releases of `iron-list` seem to break it within `paper-dialog`. We reverted it therefore to `dom-repeat`. Could you please check if demo works now for you?
We created a new release with the fix (version 1.0.20).

**Comment by cutout33** [comment]
The demo is working fine on chrome, opera, firefox, safari and edge!
I'll get the latest release and try it in my project!

thanks :+1:
